src/main/java/ymfc/commands/SortIngredientsCommand.java

```diff
@@ -0,0 +1,36 @@
+package ymfc.commands;
+
+import ymfc.list.IngredientList;
+import ymfc.list.RecipeList;
+import ymfc.storage.Storage;
+import ymfc.ui.Ui;
+
+import java.io.IOException;
+import java.util.logging.Level;
+
+import static ymfc.YMFC.logger;
+
+public class SortIngredientsCommand extends Command{
+    public static final String USAGE_EXAMPLE = """
+            Use example:
+            \tsortI
+            """;
+
+    public SortIngredientsCommand() {
+        super();
+    }
+
+
+    public void execute(RecipeList recipes, IngredientList ingredients, Ui ui, Storage storage) {
+        logger.log(Level.FINEST, "Executing SortIngredientsCommand");
+
+        ingredients.sortAlphabetically();
```

Comment:
I don't see the new `sortAlphabetically()` method in your changes. Was it previously added by someone before?

Reply:
It was already copied over by @gskang-22 from my sort recipes function, back when he was setting up the ingredients class.